plugins/by-name/quarto/default.nix

```diff
@@ -0,0 +1,114 @@
+{
+  lib,
+  pkgs,
+  helpers,
+  ...
+}:
+with lib;
+with helpers.neovim-plugin;
```

Comment:
The usage of `with` is discouraged as it can lead to non-intuitive behaviors.
Prefer a `let`-`in` block.

Have a look at [this example](https://github.com/nix-community/nixvim/blob/aabbd60633947baba11db44df84f402edc241440/plugins/by-name/notebook-navigator/default.nix#L2-L6).

Reply:
Thank you for the reference! I have updated the convention.